### ISSUE TITLE
Upload data/metadata

### ISSUE THREAD

**Issue by hkincaid** [EDRN/labcas-ui#8]
Allow user to upload data if size of data is not too big. This could be broken into 2 pieces if easier. Have a metadata form a user can complete and then have them upload? Or combine.

**Comment by nutjob4life** [comment]
@hoodriverheather this may overlap in some ways with the [data submission procedure](https://edrn.nci.nih.gov/data-and-resources/informatics/labcas-help/new-data-upload-procedure/) and the [data submission form](https://edrn.nci.nih.gov/data-and-resources/common-data-elements/metadata-collection-form/)

Thoughts? Do you want to close this issue or narrow the scope of it to data uploading?

**Comment by hoodriverheather** [comment]
@nutjob4life Good point. I know it's not a high priority right now. Lets just leave this for now and can discuss at a later date.
